charts/matrix-server/templates/statefulset.yaml

```diff
@@ -90,6 +90,11 @@ spec:
           - configMap:
               name: matrix-server-antiscam-init
           {{ end }}
+      {{ if .Values.antiscam }}
+      - name: matrix-antiscam
+        image: {{ .Values.antiscam_image.repo }}:{{ .Values.antiscam_image.tag }}
```

Comment:
We need to define default values for this in `values.yaml`

Reply:
I was going to put those in the community-matrix-server repo, or would you prefer it defined here?

Reply:
We need some default values defined here, if not helm lint will fail and the chart won't be published.

Reply:
Looking ok now?

Reply:
indeed thx!